README.md

```diff
@@ -209,6 +209,28 @@ Authorization in `root:redhat-appstudio` and `root:redhat-hacbs` workspaces in C
 
 For access to the OpenShift staging cluster, the user must be added to the `stage` team in the `redhat-appstudio-sre` Github organization.
 
+## Enabling Monitoring for workload cluster
+
+Prometheus Operator is deployed through ArgoCD. This is configure to run in `appstudio-workload-monitoring` namespace. 
+This prometheus is accessed through exported route, which is secured by github oauth. Github oauth secrets are configured in the cluster through secret
+
+`prometheus-proxy-config`
```

Comment:
will we have a script similar to https://github.com/redhat-appstudio/infra-deployments/blob/pre-kcp/hack/setup_observability.sh to create all the secrets? (ie, out of GitOps perimeter)

Reply:
like for all AppStudio secrets in general? No, it's hard to use something generic because the component's secrets are owned & managed by the corresponding teams. 
For the monitoring secrets, feel free to use whatever is easy for you.

Reply:
ok, I'll probably add a new `hack/setup-monitoring.sh` script for now, then